setup.py

```diff
@@ -94,6 +94,7 @@
         "singledispatchmethod; python_version < '3.8.0'",
         "docstring-parser>=0.9.0",
         "diskcache>=5.2.1",
+        "warlock>=1.3.3",
```

Comment:
Do we want to add one more dependency?

Reply:
What I mean - https://pypi.org/project/warlock/ - no python 3.8?

Reply:
Maybe I can try to implement it by myself.
https://github.com/bcwaldon/warlock/blob/6f41194ab675efa451470f8abfebfa6bc72403fe/warlock/core.py#L23